README.md

```diff
@@ -4,37 +4,29 @@ Native desktop application for [Mattermost](http://www.mattermost.org/) running
 
 Originally created as "electron-mattermost" by Yuya Ochiai. Developed using [Electron](http://electron.atom.io/).
```

Comment:
Can you please add the following befor this link?

[Mattermost](https://mattermost.com) is an open source platform for secure collaboration across the entire software development lifecycle. This repo is for the native desktop application that's built on [Electron](http://electron.atom.io/); it runs on Windows, Mac, and Linux.